app/models/game.rb

```diff
@@ -1,7 +1,12 @@
 class Game < ActiveRecord::Base
-  has_many :pieces, dependent: :destroy
-  validates :game_name, :presence => { :message => "Game name is required!" }
-  after_create :populate_board!
+
+	has_many :pieces, dependent: :destroy
+	has_many :users, :through => :pieces
```

Comment:
I'm not sure if this is the ideal way to do this (though I could be wrong). If you try Game.first.users with this setup, you get 32 user objects returned by the query, since there are 32 pieces that each have a user. Granted you could write some custom method to only list unique users (or they're may be a default method already in Rails), but that seems like unnecessarily adding complexity. Two other options could be to have a "GameInvitation" model with a player1 and player2 and game field to use the :through relationship for games and user (since there will only be 1 invitation per game), OR you could use "has_and_belongs_to_many" to link games and users directly. Let me know what you think.